pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -363,6 +363,10 @@ func (ic *ifConfigurator) removeHNSEndpoint(endpoint *hcsshim.HNSEndpoint, conta
 	return nil
 }
 
+func isValidHostNamespace(hostNamespace string) bool {
+	return hostNamespace != "" && hostNamespace != "00000000-0000-0000-0000-000000000000"
```

Comment:
In which case hostNamespace will be "00000000-0000-0000-0000-000000000000"? Does it apply to all Pods we create or it's because the pod is removed in some speicic order? 
Is it safe to not call `RemoveNamespaceEndpoint` at all in this case?

Reply:
"00000000-0000-0000-0000-000000000000" is used if we use "docker" to create container on Windows. It is because docker doesn't create host namespace when creating the sandbox container. But for "containerd", creating host namespace is a step before calling CNI to configure container network, so there should be a valid value.
To support containerd, we have a check here to ensure we could remove the HcnEndpoint(HnsEndpoint) from the host namespace. 
However, if we remove HcnEndpoint from host namespace "00000000-0000-0000-0000-000000000000", Windows HCN API might return an Error, and it will break the following step (deleting HndEndpoint), and cause the configuration left on the host.

Reply:
Thanks for the explanation. could we also add the comment to the code for future reference? and maybe add the runtime information to https://github.com/antrea-io/antrea/issues/2301 so people know it's specific to docker.